.flake8

```diff
@@ -8,5 +8,5 @@ extend-ignore = E203
 count = true
 max-complexity = 10
 # black's max-line-length is 89, but it doesn't touch long string literals.
-max-line-length = 100
+max-line-length = 120
```

Comment:
Wouldn't it make more sense to shorten the one overly long line in `src/antsibull_docs/write_docs/plugins.py:171` instead of raising max-line-length to 120?

Reply:
It would make more sense if black would stop artifically making this line longer than necessary by changing
```
        edit_on_github_url = (
            f"https://github.com/{eog.repository}/edit/{eog.branch}/{eog.path_prefix}{gh_path}"
        )
```
(less than 100 chars) back to 
```
        edit_on_github_url = f"https://github.com/{eog.repository}/edit/{eog.branch}/{eog.path_prefix}{gh_path}"
```
I've now split it up into even more lines so black won't screw it up.